src/azure-cli/azure/cli/command_modules/profile/__init__.py

```diff
@@ -40,43 +39,67 @@ def load_command_table(self, args):
+                            'secret or a PEM file with key and public certificate. Will prompt if not given.')
+            c.argument('tenant', options_list=['--tenant', '-t'], validator=validate_tenant,
+                       help='The Microsoft Entra tenant, must provide when using a service principal.')
+            c.argument('scopes', options_list=['--scope'], nargs='+',
+                       help='Used in the /authorize request. It can cover only one static resource.')
+            c.argument('allow_no_subscriptions', action='store_true',
+                       help="Support accessing tenants without subscriptions. It's uncommon but useful to run "
+                            "tenant-level commands, such as 'az ad'.")
             c.ignore('_subscription')  # hide the global subscription parameter
-            c.argument('identity', options_list=('-i', '--identity'), action='store_true', help="Log in using the Virtual Machine's identity", arg_group='Managed Service Identity')
-            c.argument('identity_port', type=int, help="the port to retrieve tokens for login. Default: 50342", arg_group='Managed Service Identity')
+
+            # Device code flow
             c.argument('use_device_code', action='store_true',
-                       help="Use CLI's old authentication flow based on device code. CLI will also use this if it can't launch a browser in your behalf, e.g. in remote SSH or Cloud Shell")
-            c.argument('use_cert_sn_issuer', action='store_true', help='used with a service principal configured with Subject Name and Issuer Authentication in order to support automatic certificate rolls')
-            c.argument('scopes', options_list=['--scope'], nargs='+', help='Used in the /authorize request. It can cover only one static resource.')
-            c.argument('client_assertion', options_list=['--federated-token'], help='Federated token that can be used for OIDC token exchange.')
+                       help="Use device code flow. CLI will also use this if it can't launch a browser, "
+                            "e.g. in remote SSH or Cloud Shell.")
+
+            # Service principal
+            c.argument('service_principal', action='store_true',
+                       help='Log in with a service principal.')
+            c.argument('use_cert_sn_issuer', action='store_true',
+                       help='Use Subject Name + Issuer (SN+I) authentication in order to support automatic '
+                            'certificate rolls.')
+            c.argument('client_assertion', options_list=['--federated-token'],
+                       help='Federated token that can be used for OIDC token exchange.')
+
+            # Managed identity
+            c.argument('identity', options_list=('-i', '--identity'), action='store_true',
+                       help="Log in using managed identity", arg_group='Managed Identity')
```

Comment:
Changed from "Log in using the Virtual Machine's identity". CLI not only supports Virtual Machine's managed identity, but also App Service and other services' managed identities as well.

Reply:
Changed `arg_group` from "Managed Service Identity" to "Managed Identity".